src/components/themed/ModalParts.js

```diff
@@ -27,11 +28,22 @@ export function ModalCloseArrow(props: { onPress: () => void }) {
 
   return (
     <TouchableOpacity onPress={props.onPress} style={styles.closeArrow}>
-      <AntDesignIcon name="down" size={theme.rem(1.25)} color={theme.secondaryText} />
+      <FontAwesome5 name="chevron-down" size={theme.rem(1.25)} color={theme.iconTappable} />
```

Comment:
change it back

Reply:
Ignore this comment.
Julia approved this change.